### ISSUE TITLE
A11y_AzureDataStudioAAE_Sql Instance 1_Connect to Server_Keyboard: "Error: Invalid value" is not visible on the screen when we fill wrong data in username & password field.

### ISSUE THREAD

**Issue by krrish2021** [microsoft/azuredatastudio#15761]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:** #A11yMAS;#A11yTCS;#A11ySev2;#SH_AzureDataStudio_AAE_Jun2021;#SQL Azure Data Studio;#Win10;#FTP;#DesktopApp;#Keyboard;#WCAG3.3.3;#AINotInScope;

### Environment Details:
Application Name: Azure Data Studio-AAE
Windows Version: Win10

### Repro Steps:

1. Launch the application-Azure Data Studio-Insiders and install extension Azure arc and Azure data ClI
2. Connect to Azure Arc Controller
3. Tab till Azure Arc Controller tree and tab to the node present under it
4. Tab to child node "Sqlinstance1" and open manage and tab to "Connect to Server" and press enter
5. Tab to Username and Password and fill wrong information and check if the error message is shown on the screen.

### Actual:
"Error: Invalid value" is not visible on the screen when we fill wrong data in username & password field.

### Expected:
"Error: Invalid value" should be visible on the screen when we fill wrong data in username & password field.

### User Impact:
Keyboard users will get confused if the error is not visible on the screen.

**Comment by Charles-Gagnon** [comment]
@krrish2021 When a connection error occurs a toast notification pops up and is announced by the screen readers : 

![image](https://user-images.githubusercontent.com/28519865/127197337-ebeace23-4283-4ac0-9dbc-7a62255325e7.png)

This is similar behavior to how we handle errors in other parts of the app.

I'm going to close this since it seems to be working as intended - let us know if there's additional guidance on the proper way to display errors (and provide links to documentation stating the requirements)

**Comment by krrish2021** [comment]
The bug is now not reproducing the the version 20.3.5 . Please find the attachment

https://user-images.githubusercontent.com/85863543/127600338-73868838-37b5-4a4a-ba80-5c6513eacea7.mp4

**Comment by Harika1702** [comment]
GitHub Tags: #Closed;
